src/components/NoteCard.css

```diff
@@ -41,3 +42,65 @@
   right: 4px;
   justify-content: end;
 }
+.overlay {
+  position: fixed;
+  top: 0;
+  right: 0;
+  left: 0;
+  bottom: 0;
+  background-color: rgba(0, 0, 0, 0.5);
+  z-index: 10;
+}
+.edit-note-container {
+  background: var(--white);
+  /* position: absolute; */
```

Comment:
again remove extra comments

Reply:
Thank you for deep analysis, will remove once I complete P0 features.